scipy/optimize/_minimize.py

```diff
@@ -31,18 +31,20 @@
 from .tnc import _minimize_tnc
 from .cobyla import _minimize_cobyla
 from .slsqp import _minimize_slsqp
+from ._direct import _minimize_direct
```

Comment:
I like the idea of exposing optimizers via `minimize`, but I don't have much to say about it. I suspect that having a dedicated function and not exposing it in `minimize` is the path of least resistance, though, so I'd understand if this PR goes that way.

Reply:
> I suspect that having a dedicated function and not exposing it in `minimize` is the path of least resistance, though, so I'd understand if this PR goes that way.

I believe we should have a dedicated function for direct because, it's working is different than other algorithms under `minimize`. One example is that DIRECT doesn't require initialisation guesses, it works without them but other algorithms require it. Calling, `minimize(f, None, method='direct')` would a bit un-easy and semantically holds little meaning.